site/docs/components/drawer/examples.mdx

```diff
@@ -8,35 +8,47 @@ data:
 ---
 
 <LivePreviewControls>
-<LivePreview componentName="drawer" exampleName="TopPositioned" >
+<LivePreview componentName="drawer" exampleName="DefaultPrimary" >
 
-### Top positioned
+### Default primary
 
-Set `position="top"` to display the drawer component at the top.
+Drawers are by default, anchored to the `left` edge of the screen. You can also place them on the `right`, `top` and `bottom` edges using `position` prop.
 
 </LivePreview>
 
-<LivePreview componentName="drawer" exampleName="RightPositioned" >
+<LivePreview componentName="drawer" exampleName="DefaultSecondary" >
 
-### Right positioned
+### Default secondary
+
+Set `variant="secondary"` for secondary drawer.
+
+</LivePreview>
+
+<LivePreview componentName="drawer" exampleName="RightWithFormField" >
+
+### Right drawer with form field
 
 Set `position="right"` to display the drawer component on the right.
 
+#### Best practices
+
+Consider using a right-anchored drawer to show supporting information that’s related to the current context, such as recent events, notifications or quick settings.
+
 </LivePreview>
 
-<LivePreview componentName="drawer" exampleName="LeftPositioned" >
+<LivePreview componentName="drawer" exampleName="TopWithFormField" >
 
-### Left positioned
+### Top drawer with form field
 
-By default, the drawer is displayed on the left if the `position` prop is not set.
+Set `position="top"` to display the drawer component at the top. Anchor your drawer to the top edge if it will display supporting information that’s related to the current context and is best presented in a horizontal format, e.g., complex filters or a summary panel.
```

Comment:
I find it confusing that we're spreading out best practices for drawer based on the position in multiple stories. Positions are mentioned in the "Default" example and then their best practices are in separate examples.

Reply:
revamped the examples a little, they now are categorised. In Default, there's primary and secondary. In Position, there's top/bottom/left/right. Further examples showcase more detailed/recommended usage of drawer in respective position. Hope this helps

Reply:
i have made some changes/updates to how examples (& respective descriptions) are shown on site, please hav e a look at my latest commit @mikearildbrown @joshwooding